lib/ViewModels/LocationBarViewModel.js

```diff
@@ -117,11 +135,27 @@ function updateCoordinatesFromLeaflet(viewModel, mouseMoveEvent) {
 }
 
 function cartographicToFields(viewModel, coordinates, errorBar) {
-    viewModel.latitude = Math.abs(CesiumMath.toDegrees(coordinates.latitude)).toFixed(3) + '°' + (coordinates.latitude < 0.0 ? 'S' : 'N');
-    viewModel.longitude = Math.abs(CesiumMath.toDegrees(coordinates.longitude)).toFixed(3) + '°' + (coordinates.longitude < 0.0 ? 'W' : 'E');
+
+    var zone = 1 + Math.floor((coordinates.longitude+180)/6);
```

Comment:
Also, if you profile while moving the mouse around quickly, a not insignificant amount of time is spent in `proj4`.  Can we skip the projection when only showing longitude and latitude?